app/src/main/res/layout/screen_manual_patient_entry.xml

```diff
@@ -116,6 +116,7 @@
         android:orientation="vertical">
 
         <TextView
+          android:id="@+id/textIdentifierType"
```

Comment:
Can we rename it to `identifierTypeLabel` or `identifierTypeTextView` or something to maintain code consistency please?